src/components/UserProfile/UserSettings/UserNotificationSettings/index.tsx

```diff
@@ -19,6 +19,15 @@ const messages = defineMessages({
   discordIdTip:
     'The <FindDiscordIdLink>ID number</FindDiscordIdLink> for your Discord user account',
   validationDiscordId: 'You must provide a valid Discord user ID',
+  telegramChatId: 'Telegram Chat ID',
+  telegramChatIdTip:
+    'The Chat ID can be aquired by adding <GetIdBotLink>@get_id_bot</GetIdBotLink> to the chat.',
+  telegramChatIdTipLong:
+    'Start a chat by clicking <TelegramBotLink>here</TelegramBotLink>.\
+    Then get the group Chat ID by adding <GetIdBotLink>@get_id_bot</GetIdBotLink> to that chat and send /my_id to the chat',
+  sendSilently: 'Send Silently',
+  sendSilentlyDescription: 'Send telegram notifications silently',
+  validationTelegramChatId: 'You must provide a valid Telegram Chat ID',
```

Comment:
```suggestion
  telegramChatId: 'Chat ID',
  telegramChatIdTip:
    'Can be found by adding <GetIdBotLink>@get_id_bot</GetIdBotLink> to the chat',
  telegramChatIdTipLong:
    '<TelegramBotLink>Start a chat</TelegramBotLink>, add <GetIdBotLink>@get_id_bot</GetIdBotLink>, and issue the /my_id command',
  sendSilently: 'Send Silently',
  sendSilentlyDescription: 'Send notifications with no sound',
  validationTelegramChatId: 'You must provide a valid chat ID',
```

We probably just need a separate section for these Telegram settings, and `telegramChatIdTipLong` really feels like it belongs in a info box/bubble like in Settings >Notifications.  I think it is too long to be a form label tip.

Otherwise, we can't have a field just named "Send Silently" (and "Telegram Send Silently" / "Send Telegram Silently" just don't sound right), even if there's a tip that explains it is for Telegram.  Perhaps we can use the same formatting/layout from the system-wide/admin notifications?  I imagine that this page will get kinda unwieldy as we add user-specific notification support for other agents if we don't add that type of tab structure to the page.

Reply:
We can work on refactoring the display of the user notification settings to look more like the notification settings for the app in the future. For this PR's purposes, this is fine.